test/CoinbaseSmartWallet/IsValidSignature.t.sol

```diff
@@ -165,4 +165,21 @@ contract TestIsValidSignature is SmartWalletTestBase {
         vm.expectRevert();
         account.isValidSignature(hash, abi.encode(CoinbaseSmartWallet.SignatureWrapper(1, signature)));
     }
+
+    // @dev this case should not be possible, but we need to explicitly test the revert case
```

Comment:
Does beg the question of whether we should even have this check 🤔 it predates our more opinionated addOwner methods.

Reply:
I agree that this isn't a terribly useful test today. 

Though, like it says in the comments of the contract-under-test:
```
                // technically should be impossible given owners can only be added with
                // addOwnerAddress and addOwnerPublicKey, but we leave incase of future changes.
```
 we check it to make sure for future versions. I don't think it can hurt to have the check (esp. once we have coverage reporting as part of CI).